src/gui/hotkeyConfig.cpp

```diff
@@ -343,3 +368,31 @@ void HotkeyConfigItem::load(string key_config)
         }
     }
 }
+
+bool HotkeyConfig::setHotKey(std::string work_cat, std::pair<string,string> key, string new_value)
+{
+
+    // test if new_value is part of the sfml_list
+    for (std::pair<string, sf::Keyboard::Key> sfml_key : sfml_key_names) {
```

Comment:
Nitpicking: { should be on newline

(yeah yeah, I know, it's not super consistent now, but the more we try and keep it in order, the better it will get :))